tools/setup/emoji/emoji_names.py

```diff
@@ -1518,6 +1518,8 @@
     "25fb": {"canonical_name": "white_medium_square", "aliases": []},
     "2b1b": {"canonical_name": "black_large_square", "aliases": []},
     "2b1c": {"canonical_name": "white_large_square", "aliases": []},
+    "1f7e8": {"canonical_name": "large_yellow_square", "aliases": []},
+    "1f7e9": {"canonical_name": "large_green_square", "aliases": []},
```

Comment:
Any reason for the inconsistent word order?

Reply:
These names are the same as what is defined in `emoji-datasource-twitter` and `emoji-datasource-google`. We also have `large_{blue,orange}_diamond` defined in our names. I wasn't sure what our approach with these names was, here.

Reply:
@andersk @timabbott Should we add names consistent with the white and black emojis as the canonical names and change the other ones to be aliases? Or is this good enough, since we are consistent with the Unicode standard?

Reply:
I think probably we should use the same word order present in this file; the color is much more likely to be what someone is thinking when doing typeahead than the word "large.

Reply:
Makes sense for these new emojis. I'll submit a PR to change these names. 
There are also a few other names - large and small diamonds which are named inconsistently. I was wondering what to do about them.

Reply:
> I'll submit a PR to change these names.

Thanks! For posterity, this ended up being #21031.